src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextFieldTest.java

```diff
@@ -0,0 +1,156 @@
+package org.symphonyoss.symphony.messageml.elements.form;
+
+import org.junit.Test;
+import org.symphonyoss.symphony.messageml.elements.*;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class TextFieldTest extends ElementTest {
+
+  private static final String NAME_ATTR = "name";
+  private static final String REQUIRED_ATTR = "required";
+  private static final String PLACEHOLDER_ATTR = "placeholder";
+
+
+  @Test
+  public void testTextField() throws Exception {
+    String name = "text-field";
+    String input = "<messageML><form><text-field name=\"" + name + "\"/></form></messageML>";
+    context.parseMessageML(input, null, MessageML.MESSAGEML_VERSION);
+
+    Element messageML = context.getMessageML();
+    Element form = messageML.getChildren().get(0);
+    Element textField = form.getChildren().get(0);
+
+    assertEquals("TextField class", Form.class, form.getClass());
```

Comment:
If we're going to include a message, make one that identifies the specific assert. Since this is unit test I would chose one of the two:

1 - Include a more specific message.
OR
2 - Remove the message from the assert.

Having only the "TextField class" doesn't help much here, I think.

PS: Assume this coment is applicable on all asserts with message on your unit tests in this PR.